frontend/tests/ui/test_course.py

```diff
@@ -91,15 +91,20 @@ def test_create_identities_for_course(driver):
     password = 'p@ssw0rd'
     driver.register_user(username, password)
 
+    # Admin creates a course with one assignment and one problem, and then
+    # creates some identities associated with the course group
     with driver.login_admin():
         create_course(driver, course_alias, school_name)
         add_assignment(driver, assignment_alias)
         add_problem_to_assignment(driver, assignment_alias, problem)
-        identities = util.create_identities_course(driver, course_alias)
-        (unassociated_identity, unassociated_password) = identities[0]
-        (associated_identity, associated_password) = identities[1]
+        unassoc, assoc, _, _, _ = util.add_identities_course(driver,
```

Comment:
aquí puedes hacer

```suggestion
        unassociated, associated = util.add_identities_course(...)[:2]
```

para no tener que usar tanto `_`.

Reply:
listo